src/endpoints/resources.js

```diff
@@ -1,18 +1,20 @@
 import express from 'express'
 import { ttlFromJsonld, n3FromJsonld } from '../rdf.js'
 import _ from 'lodash'
-import { handleError } from '../error.js'
 
 const resourcesRouter = express.Router()
 
 const apiBaseUrl = process.env.API_BASE_URL
 
-resourcesRouter.post('/:resourceId', (req, res) => {
+// This regex path will match legacy uris like http://localhost:3000/repository/pcc/3a941f1e-025f-4a6f-80f1-7f23203186a2
```

Comment:
This isn't needed.

Reply:
The test for post? It was in your original PR, should I just remove it then?

Reply:
Yes, no longer necessary.